File: CienciasDaComputação/Maratona SBC/Testes/iniciante/iniciante.java
import java.util.Scanner;

public class iniciante{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int t = 0;
        int acc = 0;
        for(int i=0; i<4; i++){
            t = sc.nextInt();
            acc += t;
        }
        System.out.println(acc-3);
        sc.close();
    }
}

File: CienciasDaComputação/Maratona SBC/Testes/iniciante/iniciante.c
#include <stdio.h>
int main(){
    int t = 0;
    int acc = 0;
    for(int i=0; i<4; i++){
        scanf(" %d", &t);
        acc += t;
    }
    printf("%d",acc - 3);
    return 0;
}